pkg/render/intrusion_detection.go

```diff
@@ -33,7 +33,6 @@ import (
 	operatorv1 "github.com/tigera/operator/api/v1"
 	"github.com/tigera/operator/pkg/components"
 	"github.com/tigera/operator/pkg/dns"
-	"github.com/tigera/operator/pkg/render/common/elasticsearch"
 	relasticsearch "github.com/tigera/operator/pkg/render/common/elasticsearch"
```

Comment:
why change this?

Reply:
oh extra cleanup switching over to the one defined below 
` relasticsearch "github.com/tigera/operator/pkg/render/common/elasticsearch" `